### DISCUSSION TITLE
INAV Save on disarm ver7.1.2

### DISCUSSION THREAD

**Discussion by ejrwy** [iNavFlight/inav#10134] [Q&A]
If I have enabled autotune on a fixed wing airplane, will the autotune results be saved on disarm? I do have servo autotrim turned on, I am guessing that the Saved message I get when I disarm is saving the autotrim.

**Comment by MrD-RC** [comment]
If you see a `Settings saved` message. All settings are saved.

**Reply by ejrwy** [reply]
I do get a settings saved when I dissarm. Thanks! MrD-RC
